docs/bis/ec/update-koq-on-property.md

```diff
@@ -0,0 +1,186 @@
+
+## Removing KindOfQuantity from a property
+
+Removing KOQ from an [ECProperty](./ec-property.md) is supported by using `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) on the property. However, while using `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) it must have both 'From' and 'To' properties set.
+
+Additionally, the 'From' value must match the persistence unit from the old KindOfQuantity and the 'To' value must be empty.
+
+### Examples
+
+Consider following [ECProperty](./ec-property.md) having [KindOfQuantity](./kindofquantity.md) attached to it:
+
+```xml
+<KindOfQuantity typeName='MyKindOfQuantity' description='My KindOfQuantity'
+				displayLabel='My KindOfQuantity' persistenceUnit='CM' relativeError='.5'
+				presentationUnits='FT;IN;M' />
+<ECEntityClass typeName='Foo' >
+	<ECProperty propertyName='Length' typeName='double' kindOfQuantity='MyKindOfQuantity' />
+</ECEntityClass>
+```
+
+Following are some Valid and Invalid ways of using `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) to remove [KOQ](./kindofquantity.md) from the ECProperty:
+
+#### Valid `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md)
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			<From>u:CM</From>
+			<To></To>
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+#### Invalid/malformed `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md)
+
+```xml
+<ECProperty propertyName='Length' typeName='double' />
+```
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			<From>u:CM</From>
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+```xml
+<ECProperty propertyName='Length' typeName='double'>
+	<ECCustomAttributes>
+		<AllowUnitChange xmlns="SchemaUpgradeCustomAttributes.01.00.00">
+			<From>u:M</From>
+			<To></To>
+		</AllowUnitChange>
+	</ECCustomAttributes>
+</ECProperty>
+```
+
+## Replacing KindOfQuantity of a property
+
+Changing the KindOfQuantity of an ECProperty to another KindOfQuantity with a different persistence unit is allowed by using the `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) on the property. However, `AllowUnitChange` [Custom Attribute](./ec-custom-attributes.md) must have non-empty From and To properties.
+
+Additionally, the 'From' value must match the persistence unit from the old KindOfQuantity and the 'To' value must match the persistence unit from the new KindOfQuantity.
+
```

Comment:
This article should also clarify that existing values on the property will not be changed automatically upon a KOQ update.

Reply:
I agree

Reply:
now covered in my overview suggestion